src/common/ImageAvatar.js

```diff
@@ -15,14 +15,15 @@ const styles = StyleSheet.create({
 });
 
 export default({ avatarUrl, size, status, isCircular, onPress }) => {
-  const imageStyle = {
+  const touchableStyle = {
     height: size,
     width: size,
-    borderRadius: isCircular ? size / 2 : size / 8,
   };
 
+  const imageStyle = { ...touchableStyle, borderRadius: isCircular ? size / 2 : size / 8};
```

Comment:
If it is needed, another approach would be to just set Image's style to 
`style={[imageStyle, touchableStyle]}`